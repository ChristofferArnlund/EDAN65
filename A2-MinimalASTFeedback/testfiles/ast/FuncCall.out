Program
  List
    Func
      IdDecl
        Name ID="a"
        Opt
      List
      StmtBlock
        List
          FuncCallStmt
            FuncCall
              IdUse
                Name ID="printf"
              List
                IdUse
                  Name ID="x"
                IdUse
                  Name ID="y"
                Numeral NUMERAL="5"
                Mul
                  Numeral NUMERAL="5"
                  Numeral NUMERAL="3"
