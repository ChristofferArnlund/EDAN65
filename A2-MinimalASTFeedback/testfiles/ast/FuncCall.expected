Program
  List
    Func
      Name ID="a"
      List
      List
        FuncCall
          Name ID="printf"
          List
            Name ID="x"
            Name ID="y"
            Numeral NUMERAL="5"
            Mul
              Numeral NUMERAL="5"
              Numeral NUMERAL="3"
